lib/gameboard.rb

```diff
@@ -0,0 +1,7 @@
+class Gameboard
+  def print_board(plays)
+    puts " #{plays[0]} | #{plays[1]} | #{plays[2]}"
+    puts " #{plays[3]} | #{plays[4]} | #{plays[5]}"
+    puts " #{plays[6]} | #{plays[7]} | #{plays[8]}"
+  end
```

Comment:
:bulb: - [ REQUIRED ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Reply:
Issue resolved. Thanks for the review!